### PR TITLE
MS-743 Do they need an interpreter? Page

### PR DESCRIPTION
**Pull request by jjrobothamho**
Do they need an interpreter? Page + acceptance tests

### CODE REVIEW

File: acceptance-test/user-pathways/util/selectors.js [hunk 1]
```diff
@@ -3,7 +3,6 @@
 const path = require('path');
 const bootstrap = require('../../bootstrap/bootstrap');
 
-const VIEWPORT = { width: 1920, height: 1080 };
```
Comment (sulthan-ahmed) [inline acceptance-test/user-pathways/util/selectors.js]: you don't need the viewport anymore?

File: apps/nrm/views/does-pv-need-support.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  {{$page-content}}
+    <p>{{#t}}pages.does-pv-need-support.paragraph-1{{/t}}</p>
+    <ul class="govuk-list govuk-list--bullet">
+        <li>{{#t}}pages.does-pv-need-support.bullet-1{{/t}}</li>
+        <li>{{#t}}pages.does-pv-need-support.bullet-2{{/t}}</li>
+        <li>{{#t}}pages.does-pv-need-support.bullet-3{{/t}}</li>
+        <li>{{#t}}pages.does-pv-need-support.bullet-4{{/t}}</li>
+        <li>{{#t}}pages.does-pv-need-support.bullet-5{{/t}}</li>
+    </ul>
+    <p><a href="#">{{#t}}pages.does-pv-need-support.link-text-1{{/t}}</a> {{#t}}pages.does-pv-need-support.link-text-2{{/t}}</p>
```
Comment (sulthan-ahmed) [inline apps/nrm/views/does-pv-need-support.html]: can't this all be done in Markdown? not going to hold up the PR for that

File: apps/nrm/views/partials/pv-nationality-second.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<details class="govuk-details" role="group">
+    <summary class="govuk-details__summary" role="button" aria-controls="details-content-1da772bb-ab6c-4de6-bcad-01d16cbaa0c5" aria-expanded="true">
```
Comment (sulthan-ahmed) [inline apps/nrm/views/partials/pv-nationality-second.html]: what is `details-content-1da772bb-ab6c-4de6-bcad-01d16cbaa0c5`?

File: apps/nrm/views/types-of-exploitation.html [hunk 4]
```diff
@@ -1,49 +1,49 @@
...
-        <span class="govuk-warning-text__assistive">Warning</span>
-         If you think they are a victim of another crime you can report it to the police by calling 101.
+        <span class="govuk-warning-text__assistive">{{#t}}pages.types-of-exploitation.warning-heading{{/t}}</span>
+        {{#t}}pages.types-of-exploitation.warning{{/t}}
       </strong>
     </div>
     {{#input-submit}}continue{{/input-submit}}
   {{/page-content}}
-{{/partials-page}}
+{{/partials-page}}
```
Comment (sulthan-ahmed) [inline apps/nrm/views/types-of-exploitation.html]: Missing line?

File: acceptance-test/user-pathways/util/selectors.js (hunk 1 above)
Comment (jjrobothamho) [reply acceptance-test/user-pathways/util/selectors.js]: this is now in browser.js

File: apps/nrm/views/partials/pv-interpreter-requirements-language.html [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<fieldset id="pv-interpreter-requirements-language-fieldset" class="panel-indent">
+        {{#renderField}}pv-interpreter-requirements-language{{/renderField}}
+ </fieldset>
```
Comment (sulthan-ahmed) [inline apps/nrm/views/partials/pv-interpreter-requirements-language.html]: Too many spaces? Can it be two spaces?

**Comment by sulthan-ahmed** [comment]
Looks good

File: acceptance-test/user-pathways/util/selectors.js (hunk 1 above)
Comment (sulthan-ahmed) [reply acceptance-test/user-pathways/util/selectors.js]: 👍

**Pull request merged**